afkak/kafkacodec.py

```diff
@@ -32,7 +32,7 @@
 from .common import (
     CODEC_GZIP, CODEC_NONE, CODEC_SNAPPY, BrokerMetadata, BufferUnderflowError,
     ChecksumError, ConsumerFetchSizeTooSmall, ConsumerMetadataResponse,
-    FetchResponse, InvalidMessageError, LeaveGroupResponse, Message,
+    FetchResponse, InvalidMessageError, _LeaveGroupResponse, Message,
```

Comment:
Seems like this change should be made throughout and include 'LeaveGroupRequest' as well?

Reply:
Yeah, I pushed halfway through the changeset... sorry for the inconsistent view.